### ISSUE TITLE
Still getting hack detected.

### ISSUE THREAD

**Issue by trextommy** [Exitium-DEV/HD2-Cheat-Launcher#8]
using modded game launcher and newest DLL. i swear yall need to make a discord server to manage this. some of us aint too bright lmao

**Comment by xicemanx117** [comment]
Got hack Detection also like as soon as i started a mission

**Comment by Speedsterfun** [comment]
> using modded game launcher and newest DLL. i swear yall need to make a discord server to manage this. some of us aint too bright lmao

Yeah, a discord would be nice, as I don't know what I am doing half the time.

Also, yeah, it is getting detected for me too

**Comment by isaaclid6** [comment]
same to me, got detected in 5 minutes after game launchhh

**Comment by trextommy** [comment]
> Compile your own emoisback DLL, change the strings in the DLL to whatever you want (just not the defaults), then use that DLL with the launcher, it works

how

**Comment by TakiDn** [comment]
> Compile your own emoisback DLL, change the strings in the DLL to whatever you want (just not the defaults), then use that DLL with the launcher, it works

How do you change strings ?

**Comment by TakiDn** [comment]
> > Compile your own emoisback DLL, change the strings in the DLL to whatever you want (just not the defaults), then use that DLL with the launcher, it works
> 
> How do you change strings ?

Did all the above beside changing strings.
